src/sqlfluff/rules/L001.py

```diff
@@ -30,6 +30,7 @@ class Rule_L001(BaseRule):
         FROM foo
     """
 
+    groups = ("all", "core")
```

Comment:
What's the benefit of having an "all" group?

Reply:
I think if in the future we only enable core rules by default, this will give users the option to turn on all the rules with `--rules all`

Reply:
Ah, good point!

Reply:
ON that, should we update `default_config` to be `all` rather than the current `None`:

https://github.com/sqlfluff/sqlfluff/blob/b31402765168b65841165b47a3d76b509c52a64d/src/sqlfluff/core/default_config.cfg#L11-L12

We can leave `none` to work as it currently does (i.e. being same as `all`) but putting `all` in our default config would be clearer IMHO.